modulefiles/ufs_common.lua

```diff
@@ -4,23 +4,23 @@ help([[Load UFS Model common libraries]])
 
 local ufs_modules = {
   {["jasper"]          = "2.0.32"},
-  {["zlib"]            = "1.2.13"},
+  {["zlib-ng"]         = "2.1.6"},
   {["libpng"]          = "1.6.37"},
-  {["hdf5"]            = "1.14.0"},
+  {["hdf5"]            = "1.14.3"},
   {["netcdf-c"]        = "4.9.2"},
   {["netcdf-fortran"]  = "4.6.1"},
-  {["parallelio"]      = "2.5.10"},
-  {["esmf"]            = "8.6.0"},
-  {["fms"]             = "2024.01"},
+  {["parallelio"]      = "2.6.2"},
+  {["esmf"]            = "8.6.1"},
+  {["fms"]             = "2024.02"},
   {["bacio"]           = "2.4.1"},
-  {["crtm"]            = "2.4.0"},
+  {["crtm"]            = "2.4.0.1"},
   {["g2"]              = "3.5.1"},
   {["g2tmpl"]          = "1.13.0"},
-  {["ip"]              = "4.3.0"},
+  {["ip"]              = "5.0.0"},
   {["sp"]              = "2.5.0"},
   {["w3emc"]           = "2.10.0"},
-  {["gftl-shared"]     = "1.6.1"},
-  {["mapl"]            = "2.40.3-esmf-8.6.0"},
+  {["gftl-shared"]     = "1.9.0"},
+  {["mapl"]            = "2.46.3-esmf-8.6.1"},
```

Comment:
FYI. MAPL 2.46.3 does not work in UFS WM.

Reply:
@junwang-noaa , yes, eventually we will have to create new environment with MAPL version that will work for WM.